packages/synapse-types/src/File/FileHandle.ts

```diff
@@ -27,4 +30,6 @@ export interface FileHandle {
   storageLocationId: number
   /** The size of the file in bytes. */
   contentSize: number
+  /** The status of the file handle as computed by the backend. This value cannot be changed, any file handle that is not AVAILABLE should not be used. */
+  status: FileHandleStatus
```

Comment:
Some mock data had these new properties, so they needed to be added to our types.